client/src/components/geneExpression/infoPanel/common/infoPanelContainer/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { kebabCase } from "lodash";
+import { InfoContainer, InfoWrapper } from "../style";
+import {
+  ErrorInfo,
+  LoadingInfo,
+  NoneSelected,
+  ShowInfo,
+} from "../infoPanelParts";
+import { ExtendedInfoProps } from "../types";
+
+function ContainerInfo(props: ExtendedInfoProps) {
```

Comment:
Common container for either Gene Info or Cell Type Info

Reply:
Great extraction 👏